tests/functional/tnf/test_tnf_deployment.py

```
@@ -0,0 +1,570 @@
+from ocs_ci.utility.reporting import get_polarion_id
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import ceph_health_check, is_cluster_running
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+logger = logging.getLogger(__name__)
+
+
+@turquoise_squad
+@deployment
+@polarion_id(get_polarion_id())
+def test_tnf_deployment(pvc_factory, pod_factory):
```

Ruff (0.16.2)

[warning] Unused function argument: `pvc_factory`

(ARG001)